Godeps/_workspace/src/k8s.io/kubernetes/pkg/kubectl/describe.go

```diff
@@ -72,18 +73,19 @@ func (e ErrNoDescriber) Error() string {
 
 func describerMap(c *client.Client) map[unversioned.GroupKind]Describer {
 	m := map[unversioned.GroupKind]Describer{
-		api.Kind("Pod"):                   &PodDescriber{c},
-		api.Kind("ReplicationController"): &ReplicationControllerDescriber{c},
-		api.Kind("Secret"):                &SecretDescriber{c},
-		api.Kind("Service"):               &ServiceDescriber{c},
-		api.Kind("ServiceAccount"):        &ServiceAccountDescriber{c},
-		api.Kind("Node"):                  &NodeDescriber{c},
-		api.Kind("LimitRange"):            &LimitRangeDescriber{c},
-		api.Kind("ResourceQuota"):         &ResourceQuotaDescriber{c},
-		api.Kind("PersistentVolume"):      &PersistentVolumeDescriber{c},
-		api.Kind("PersistentVolumeClaim"): &PersistentVolumeClaimDescriber{c},
-		api.Kind("Namespace"):             &NamespaceDescriber{c},
-		api.Kind("Endpoints"):             &EndpointsDescriber{c},
+		api.Kind("Pod"):                        &PodDescriber{c},
+		api.Kind("ReplicationController"):      &ReplicationControllerDescriber{c},
+		api.Kind("Secret"):                     &SecretDescriber{c},
+		api.Kind("Service"):                    &ServiceDescriber{c},
+		api.Kind("ServiceAccount"):             &ServiceAccountDescriber{c},
+		api.Kind("Node"):                       &NodeDescriber{c},
+		api.Kind("LimitRange"):                 &LimitRangeDescriber{c},
+		api.Kind("ResourceQuota"):              &ResourceQuotaDescriber{c},
+		api.Kind("PersistentVolume"):           &PersistentVolumeDescriber{c},
+		api.Kind("PersistentVolumeClaim"):      &PersistentVolumeClaimDescriber{c},
+		api.Kind("Namespace"):                  &NamespaceDescriber{c},
+		api.Kind("Endpoints"):                  &EndpointsDescriber{c},
+		api.Kind("SecurityContextConstraints"): &SecurityContextConstraintsDescriber{c},
```

Comment:
that's how they've been for a while.  There is a special case for them in rest mapping for when the singular == the plural.